src/__tests__/components/SeasonCard.test.tsx

```diff
@@ -0,0 +1,79 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { SeasonCard } from '../../components/';
+import { SEASON } from '../constants';
+import { MemoryRouter, Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+import { vi } from 'vitest';
+
+const TMDB_BASE_PATH = 'https://image.tmdb.org/t/p/w500';
```

Comment:
nit: We have 4 instances of this variable that are all the same. I wouldn't expect it to change, but it still might be nice to have it defined in a central location. I think https://github.com/Thenlie/Streamability/blob/main/src/helpers/constants.ts would make the most sense.